database/migrations/2019_02_05_084300_add_acceptance_id_to_submissions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddAcceptanceIdToSubmissions extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('submissions', function (Blueprint $table) {
+            $table->string('acceptance_id', 36)->nullable();
```

Comment:
Interesting.. I sort of think of the acceptance as belonging to the submission instead of the other way around. What do you think?

Reply:
I was thinking that most often, you'd want to know if a submission was accepted, rather than what submission an acceptance came from. 

It allows for easy scopes on submissions for `where acceptance_id is not null`, without having to join to acceptances table or do multiple queries. It's easy enough to change if you feel strongly in the other direction, though.

Reply:
Makes sense! Let's try it :)